components/GlobalStyle/GlobalStyle.ts

```diff
@@ -244,10 +248,23 @@ a:not([class]) {
     --text-xs: ${textXs};
     --text-fine: ${textFine};
     /* Containers */
-    --desktop-max-width: ${desktopMaxWidth}px;
-    --desktop-panel-width: ${desktopPanelWidth}px;
-    --header-height: ${headerHeight}px;
-    --banner-height: ${bannerHeight}px;
+    --mobile-page-width: ${mobilePageWidth};
+    --desktop-page-width: ${desktopPageWidth}px;
+    --mobile-header-height: ${mobileHeaderHeight}px;
+    --desktop-header-height: ${desktopHeaderHeight}px;
+    --mobile-banner-height: ${mobileBannerHeight}px;
+    --desktop-banner-height: ${desktopBannerHeight}px;
+    --page-width: var(--desktop-page-width);
+    --header-height: var(--desktop-header-height);
+    --banner-height: var(--desktop-banner-height);
+    @media ${tabletAndUnder} {
+      --page-width: var(--mobile-page-width);
```

Comment:
we set the global container dimensions at the highest level so the rest of the components don't need to think about it